napari/layers/_tests/test_layer_actions.py

```diff
@@ -86,3 +87,24 @@ def test_convert_dtype(mode):
 
     assert ll[-1].data[5, 5] == 1000
     assert ll[-1].data.flatten().sum() == 1000
+
+
+@pytest.mark.parametrize('order', [(0, 1, 2, 3), (3, 1, 2, 0), (2, 1, 0, 3)])
+def test_split_stack(make_napari_viewer, order) -> None:
+    axis = 0
+    larger_data = np.zeros((2, 3, 4, 5), dtype=np.uint8)
+    data = np.zeros((4, 6, 8), dtype=np.uint8)
+
+    viewer = make_napari_viewer()
```

Comment:
could use `viewer = ViewerModel()` here instead... then just directly access `from napari.layers._layer_actions import _split_stack` ... no need for Qt anywhere here